src/components/ActivityModal.tsx

```
@@ -0,0 +1,201 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+import cross from "@/assets/icons/crossIcon.svg";
+import { useActivityData } from "@/hooks/useActivityData";
+import { 
+  BundleActivity, 
```

ESLint

[error] 'BundleActivity' is defined but never used.

(@typescript-eslint/no-unused-vars)